webpack.config.babel.js

```diff
@@ -43,7 +43,7 @@ export default {
     extensions: [".js", ".jsx", ".json", ".tsx", ".ts"],
   },
   devServer: {
-    contentBase: path.join(__dirname, "src"),
+    static: path.join(__dirname, "src")
```

Comment:
why did you have to make this change here?

Reply:
Because the contentBase option doesn't exist on the new version of Webpack/DevServer. They did several changes on this version, including on the settings options.